app/controllers/messages_controller.rb

```diff
@@ -3,13 +3,16 @@ class MessagesController < ApplicationController
   before_action :require_active_subscription!
 
   def create
+    @conversation = conversation
     @message = Message.new(message_params.merge(conversation:, sender:))
     authorize @message, policy_class: MessagingPolicy
 
     if @message.save
-      redirect_to conversation
+      respond_to do |format|
+        format.turbo_stream { @new_message = conversation.messages.build }
+        format.html { redirect_to conversation }
```

Comment:
I know this is a `create` request so POST requests dont fall victim to it, but I've been making it a habit of specifying `status: :see_other` or `status: 303` just in case! (this is merely personal preference)